src/QMCWaveFunctions/Jastrow/CountingFunctor.h

```diff
@@ -0,0 +1,433 @@
+
+#ifndef QMCPLUSPLUS_COUNTING_FUNCTOR_H
```

Comment:
Missing licence header. Please add and update the header at the beginning of every file you introduced.